open_schools_platform/common/services.py

```diff
@@ -96,3 +99,12 @@ def get_object_by_id_in_field_with_checks(filters, request, fields: Dict[str, Ca
             result.append(None)
 
     return result
+
+
+def convert_str_date_to_datetime(date: str, time: str):
+    """
+    This function allows to convert any format date-string
+    to datetime object
+    """
+    date = (parse(date, fuzzy=True)).strftime(f"%Y-%m-%d {time}")
+    return make_aware(datetime.strptime(date, "%Y-%m-%d %H:%M:%S"))
```

Comment:
in my mind this is util
utils are functions that we can use in other project, not only openschools
it meets the definition